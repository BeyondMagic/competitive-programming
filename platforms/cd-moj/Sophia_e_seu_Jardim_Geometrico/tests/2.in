3 6
(__)...
(.Xx)..
c(.)(.)
4
p o 2 3
p " 3 3
p " 3 6
p o 2 4
