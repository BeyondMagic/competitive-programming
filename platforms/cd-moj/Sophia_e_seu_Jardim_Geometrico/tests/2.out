2
(__)...
(.oo)..
c(")(")
